service_config.json

```diff
@@ -1,3 +1,8 @@
 {
-  "version":"0.0.1"
+  "version":"0.0.1",
```

Comment:
is there a schema begin used for the versioning? Something like https://semver.org/ ?

Reply:
That value is updated using semver.